scrapinghub/client.py

```diff
@@ -282,21 +288,34 @@ class Spider(object):
 
     Usage::
 
-        >>> project.spiders.get('spider2')
+        >>> spider = project.spiders.get('spider1')
         <scrapinghub.client.Spider at 0x106ee3748>
         >>> spider.id
-        2
+        '1'
+        >>> spider.key
+        '123/1'
         >>> spider.name
-        spider2
+        'spider1'
     """
 
     def __init__(self, client, projectid, spiderid, spidername):
         self.projectid = projectid
-        self.id = spiderid
+        self.key = '{}/{}'.format(str(projectid), str(spiderid))
+        self.id = str(spiderid)
```

Comment:
I find co-existance of id and key in Spider object a little bit confusing. Do we need them both? If so - why project and job object don't have both key and id. If id is needed only for internal use (e.g. to construct queries) - maybe we can make it private and remove from docs? If we expect users to use it - let's keep it public, but let's also be consistent.